AsyncDisplayKit/Details/ASDataController.mm

```diff
@@ -417,6 +446,9 @@ - (void)reloadSections:(NSIndexSet *)sections withAnimationOptions:(ASDataContro
       
       [_editingTransactionQueue addOperationWithBlock:^{
         NSArray *indexPaths = ASIndexPathsForMultidimensionalArrayAtIndexSet(_editingNodes, sections);
+        
+        LOG(@"Edit Transaction - reloadSections: updatedIndexPaths: %@, indexPaths: %@, _editingNodes: %@", updatedIndexPaths, indexPaths, ASIndexPathsForMultidimensionalArray(_editingNodes));
```

Comment:
I'm pretty sure the macro you wrote will precompile to "literally nothing" here, but just hoping this particular argument doesn't get executed when disabled — it would be a big performance cost if I happened to be wrong in my interpretation of the macro.